### ISSUE TITLE
Colour being set to 0 and geometry size enlarging unintentionally

### ISSUE THREAD

**Issue by benjaminchlee** [MaximeCordeil/IATK#4]
All colour values of a visualisation (rgba) get set to 0 and the geometry size increases to some inflated amount during both in editor and runtime.

This seems to happen whenever play mode is entered/exited, and also whenever a visualisation is disabled and re-enabled.

**Comment by atzanteotl** [comment]
@MaximeCordeil did you have an idea as to what causes this?

**Comment by MaximeCordeil** [comment]
@atzanteotl this is an error with the serialization and deserialization of the visualisations properties. There is a bug when relaoding savec values at init.

**Comment by atzanteotl** [comment]
I've pushed a fix for the colour issue I believe

**Comment by atzanteotl** [comment]
The size issue is peculiar. When you create a vis, it seems that maxSize has no effect on glyphs? If you reload the scene, then maxSize works as expected. 
Can someone else confirm this behaviour?

**Comment by benjaminchlee** [comment]
Sorry for the late reply, I've had this problem arise again and I think I might have figured out what causes it (at least for my specific circumstance).

I noticed that the problem only occurs when the size dimension is "Undefined". When set to anything else, the sizes of the glyphs work as expected.

In line 236 of ScatterplotVisualisation.cs, the size channel of all glyphs is set to 1 when there is no size dimension:
```
viewList[i].SetSizeChannel(Enumerable.Repeat(1f, visualisationReference.dataSource[0].Data.Length).ToArray());
```

This causes the y normals in GSBillboard.shader to be set to 1, meaning that the size of every single glyph is increased by the maximum value possible as a result of sizeFactor (hence why the resulting glyph size appears to be massive).
```
float dist = 1;
float sizeFactor = normaliseValue(p[0].normal.y, 0.0, 1.0, _MinSize, _MaxSize);
										
float halfS = 0.025f * (_Size + (dist * sizeFactor));
```

A simple fix is just to change the value from 1 to 0. This fixes my particular issue and I think should fix any other issues too.

I only tried this fix with the scatterplot visualisation and points geometry. I haven't looked at any others yet.
